public/app/features/plugins/admin/components/PluginListItem.tsx

```diff
@@ -21,8 +22,13 @@ export function PluginListItem({ plugin, pathName, displayMode = PluginListDispl
   const styles = useStyles2(getStyles);
   const isList = displayMode === PluginListDisplayMode.List;
 
+  const href =
+    plugin.type === PluginType.datasource
+      ? locationUtil.assureBaseUrl(`${CONNECTIONS_ROUTES.DataSources}/${plugin.id}`)
+      : `${pathName}/${plugin.id}`;
+
```

Comment:
The jump from plugins catalog to connections from a user perspective feels rather jarring/confusing as the side navigation and breadcrumbs change and forces the user to the browser back button as the only way to return to where they were. Is this change required to manage a datasource instance? If not I would vote that we don't add this right now. @sympatheticmoose WDYT?

Reply:
Agreed 👍 I think we still remove the duplicate pages issue without requiring this redirect at this stage, and this could add more confusion than value.

Reply:
:ok_hand: I'll revert this change